org.robiot.xclicker.yml

```diff
@@ -0,0 +1,78 @@
+app-id: org.robiot.xclicker
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: xclicker
+desktop-file-name-suffix: " (Flatpak)"
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --socket=ssh-auth
+  - --share=network
+  - --device=all
+  - --filesystem=xdg-config/gtk-3.0
+  - --filesystem=host
+  - --allow=devel
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=org.freedesktop.Flatpak
+  - --filesystem=xdg-config/kdeglobals:ro
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --talk-name=com.canonical.AppMenu.Registrar.*
+  - --system-talk-name=org.freedesktop.login1
+
+modules:
+  - name: ImageMagick
+    config-opts:
+      - --enable-static=no
+      - --with-modules
+      - --disable-docs
+      - --disable-deprecated
+      - --without-autotrace
+      - --without-bzlib
+      - --without-djvu
+      - --without-dps
+      - --without-fftw
+      - --without-fontconfig
+      - --without-fpx
+      - --without-freetype
+      - --without-gvc
+      - --without-jbig
+      - --without-jpeg
+      - --without-lcms
+      - --without-lzma
+      - --without-magick-plus-plus
+      - --without-openexr
+      - --without-openjp2
+      - --without-pango
+      - --without-raqm
+      - --without-tiff
+      - --without-webp
+      - --without-wmf
+      - --without-x
+      - --without-xml
+      - --without-zlib
+    sources:
+      - type: archive
+        url: https://imagemagick.org/download/releases/ImageMagick-7.0.10-7.tar.xz
+        sha256: 7a0365e20eeef2129cb8ffee1acf1d21cdbc2ea8b57ce2941c6ca0e935d4f843
+
+  - name: xclicker
+    buildsystem: meson
+    build-commands:
+      - . ${FLATPAK_BUILDER_BUILDDIR}/build
```

Comment:
1. there is no justification for that build script. All the commands could be here
2. this should be `post-install`
3. for the icon, the best solution would be to just have the 128x128 version checked in here. Building imagemagick for that is unecessary.

Reply:
With just the 128x128 icon in share/icons/hicolor/128x128/apps it doesn't show up everywhere.

I tested in a clean enviroment. The icon loads in the app launcher, but on the titlebar it's just a placeholder.
![image](https://user-images.githubusercontent.com/68228472/159157354-6113e887-2498-45a3-ae68-79747a679fb7.png)
![image](https://user-images.githubusercontent.com/68228472/159157427-a4fbf51a-8973-4d55-bce0-67338dbe8b67.png)